### PR TITLE
Update fr.js

### LINKED ISSUE #273: Missing translation key for "accountwizard.import"
That appeared in my browser console, when using cozy in French.

### PR DESCRIPTION
**Pull request by nicofrand**
Fix #273

### CODE REVIEW

File: client/locales/fr.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = {
     accountwizard: {
         title: 'Bienvenue !',
         content: "Kresus est un gestionnaire de finances personnelles qui vous permet de mieux comprendre quelles sont vos dépenses, en calculant des statistiques intéressantes sur vos opérations bancaires. Pour commencer, veuillez remplir le formulaire ci-dessous :",
+        import: "Si vous avez exporté votre précédente instance de Kresus, vous pouvez également l'importer de nouveau en sélectionnant le fichier JSON créé lors de l'import.
```
Comment (bnjbvr) [inline client/locales/fr.js]: The terminal double-quotes is missing. Does it build on your machine?

**Comment by nicofrand** [comment]
I don't know why I did not see it. I'll correct it, squash and rebase

**Comment by nicofrand** [comment]
That should be good now !

**Comment by bnjbvr** [comment]
Thanks, can you rebase please?

**Comment by bnjbvr** [comment]
Thank you!

**Pull request merged**